### ISSUE TITLE
[FEATURE] Support Hybrid Search in Java Client

### ISSUE THREAD

**Issue by yudhiesh** [opensearch-project/opensearch-java#737]
### Is your feature request related to a problem?
_A clear and concise description of what the problem is, e.g. I'm always frustrated when [...]._
I noticed that the Java Client does not have a way to perform a Hybrid Query leveraging Exact Keyword Search and Semantic Search as per the documentation [here](https://opensearch.org/docs/latest/query-dsl/compound/hybrid/). I don't mind working on this feature. 

### What solution would you like?
_A clear and concise description of what you want to happen._
A method for `hybrid()` within the `QueryBuilder` class where we pass a List of Queries to run:
```java
List<Query> queries = new ArrayList<>();
NeuralQuery neuralQuery = new NeuralQuery.Builder().build();
MatchQuery matchQuery = new MatchQuery.Builder().build();
queries.add(neuralQuery);
queries.add(matchQuery);
Query query = new Query.Builder()
                    .hybrid(queries)
                    .build();
```

### What alternatives have you considered?
_A clear and concise description of any alternative solutions or features you've considered._

### Do you have any additional context?
_Add any other context or screenshots about the feature request here._

**Comment by yudhiesh** [comment]
> @yudhiesh thanks for raising this, please do contribute!

Would love to, any guides on how I can contribute to the codebase for this?

**Comment by dblock** [comment]
I don't have anything specific, would dig through recent PRs for adding query builders.

**Comment by VachaShah** [comment]
@yudhiesh This feature is now released in 2.9.0 of the java client via PR https://github.com/opensearch-project/opensearch-java/pull/850. Please reopen this issue if there are any issues. Thanks!
